PlanningPoker.Tests/ServicesTests.cs

```diff
@@ -5,108 +5,155 @@
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc.Formatters.Internal;
 using Microsoft.AspNetCore.SignalR;
+using Microsoft.EntityFrameworkCore;
 using NSubstitute;
+using NSubstitute.ReceivedExtensions;
+using PlanningPoker.Context;
 using PlanningPoker.Models;
+using PlanningPoker.Repostitories;
 using PlanningPoker.Services;
 using Xunit;
 
 namespace PlanningPoker.Tests
 {
     public class ServicesTests
```

Comment:
You have services for two classes here, I would split them into two separate test classes